test/trainer/test_model_iteration_averaging.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Third-party imports
+import mxnet as mx
+import mxnet.gluon.nn as nn
+import numpy as np
+import pytest
+import pandas as pd
+import math
+
+# First-party imports
+from gluonts.dataset.common import ListDataset
+from gluonts.model.simple_feedforward import SimpleFeedForwardEstimator
+from gluonts.mx.trainer import Trainer
+from gluonts.mx.trainer.model_iteration_averaging import (
+    IterationAveragingStrategy,
+    NTA,
+    Alpha_Suffix,
+)
+
+
+def initialize_model() -> nn.HybridBlock:
+    # dummy training data
+    N = 10  # number of time series
+    T = 100  # number of timesteps
+    prediction_length = 24
+    freq = "1H"
+    custom_dataset = np.zeros(shape=(N, T))
+    start = pd.Timestamp(
+        "01-01-2019", freq=freq
+    )  # can be different for each time series
+    train_ds = ListDataset(
+        [
+            {"target": x, "start": start}
+            for x in custom_dataset[:, :-prediction_length]
+        ],
+        freq=freq,
+    )
+    # create a simple model
+    estimator = SimpleFeedForwardEstimator(
+        num_hidden_dimensions=[10],
+        prediction_length=prediction_length,
+        context_length=T,
+        freq=freq,
+        trainer=Trainer(
+            ctx="cpu", epochs=1, learning_rate=1e-3, num_batches_per_epoch=1,
+        ),
+    )
+
+    # train model
+    predictor = estimator.train(train_ds)
+
+    return predictor.prediction_net
+
+
+@pytest.mark.parametrize(
+    "n", [-2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 20]
+)
+def test_NTA_V1(n: int):
```

Comment:
It would be good to unite the `test_NTA_V1` and `test_NTA_V2` to `test_NTA` with an extra `@pytest.mark.parametrize("n_last", [True, False])` parameter since the tests are almost identical... only a couple of lines change overall.